assisted-installer/bili/README.md

```diff
@@ -0,0 +1,83 @@
+# **Deploying an OpenShift Cluster using BILI workflow**
+
+This document provides a first view at deploying a cluster using bili workflow, which is a subcommand for openshift-install that will generate a custom iso for installing openshift from a set of ZTP-like manifests and in a unattended fashion.
+
+## **Worklow**
+
+### **Requisites**
+
+#### **Packages**
+
+The openshift-install build with agent support needs to be used. In the future, this will be part of the regular install
```

Comment:
```suggestion
A special openshift-install binary built with agent support must be used. In the future, this will be part of the regular openshift-install binary.
```